### PR TITLE
Add /windows endpoint for Windows configuration + /apple endpoint cleanup

### LINKED ISSUE #340: Windows equivalent to /apple
Create a Windows equivalent to the /apple endpoint with:

- A downloadable reg file with the two keys needed: https://www.laptopmag.com/uk/articles/create-reg-file-for-registry-hacks

- Command examples: https://www.windowscentral.com/how-edit-registry-using-command-prompt-windows-10

- (bonus) Ansible: https://github.com/kradalby/plays/blob/master/roles/win_tailscale/tasks/main.yaml#L8

Pre-filled in with headscale url.

We can probably move the content of all of these from apple.go to platform.go or the likes.

The keys needed to be set is listed in the ansible example.

If anyone have examples of the regedit files and commands, that would be helpful, please post it in the comments.

### PR DESCRIPTION
**Pull request by e-zk**
- [x] read the [CONTRIBUTING guidelines](README.md#user-content-contributing)
- [x] raised a GitHub issue or discussed it on the projects chat beforehand
- [ ] added unit tests
- [ ] added integration tests
- [ ] updated documentation if needed
- [x] updated CHANGELOG.md

Closes #340.

`/windows`:  
- Allows download of pre-filled `.reg` file `/windows/tailscale.reg`
- Includes CLI/command-prompt instructions as well

`/apple` changes:  
- Renamed the function that prints HTML to `AppleConfigMessage` to match `WindowsConfigMessage`.
- The template for `/apple` used `.Url`, when the correct variable needed to be `.URL`. This has been fixed.

### CODE REVIEW

File: apple_mobileconfig.go [hunk 1]
```diff
@@ -11,26 +11,27 @@ import (
...
-		    The profiles will configure Tailscale.app to use {{.Url}} as its control server.
+		    The profiles will configure Tailscale.app to use <code>{{.URL}}</code> as its control server.
 		</p>
 
 		<h3>Caution</h3>
-		<p>You should always inspect the profile before installing it:</p>
+		<p>You should always download and inspect the profile before installing it:</p>
 		<!--
-		<p><code>curl {{.Url}}/apple/ios</code></p>
+		<pre><code>curl {{.URL}}/apple/ios</code></pre>
```
Comment (juanfont) [inline apple_mobileconfig.go:126]: Should we add a note here saying that it is not currently working?

**Review by juanfont — APPROVED** [review]

File: apple_mobileconfig.go (hunk 1 above)
Comment (kradalby) [reply apple_mobileconfig.go:126]: It is commented out, `<!--` so it does not show up on the html page

**Comment by kradalby** [comment]
Looks sensible, just need the golangcilint to pass :)

**Comment by e-zk** [comment]
@kradalby @juanfont latest commit fixed comments causing golangci-lint to fail 👍

**Review by kradalby — APPROVED** [review]

**Pull request merged**